connectors/src/connectors/zendesk/temporal/gc_activities.ts

```diff
@@ -244,20 +251,19 @@ export async function removeEmptyCategoriesActivity(connectorId: number) {
     dataSourceId: dataSourceConfig.dataSourceId,
   };
 
-  const categoryIds = (
-    await ZendeskCategoryResource.fetchIdsForConnector(connectorId)
-  ).map(({ categoryId }) => categoryId);
+  const categoryAndBrandIds =
```

Comment:
nit: this naming makes beleive that you can get ids either from categories or brands
hard to name but maybe `cateroryIdsWithBrand`

Reply:
indeed I like that better 👍

Reply:
done